testing/trino-testing/src/main/java/io/trino/testing/FaultTolerantExecutionConnectorTestHelper.java

```diff
@@ -28,8 +28,13 @@ public static Map<String, String> getExtraProperties()
                 .put("retry-initial-delay", "50ms")
                 .put("retry-max-delay", "100ms")
                 .put("fault-tolerant-execution-partition-count", "5")
-                .put("fault-tolerant-execution-target-task-input-size", "10MB")
-                .put("fault-tolerant-execution-target-task-split-count", "4")
+                .put("fault-tolerant-execution-arbitrary-distribution-compute-task-target-size-min", "5MB")
+                .put("fault-tolerant-execution-arbitrary-distribution-compute-task-target-size-max", "10MB")
+                .put("fault-tolerant-execution-arbitrary-distribution-write-task-target-size-min", "10MB")
+                .put("fault-tolerant-execution-arbitrary-distribution-write-task-target-size-max", "20MB")
+                .put("fault-tolerant-execution-hash-distribution-compute-task-target-size", "5MB")
+                .put("fault-tolerant-execution-hash-distribution-write-task-target-size", "10MB")
+                .put("fault-tolerant-execution-standard-split-size", "2.5MB")
```

Comment:
Commit boundary is messed up here. The `Add an explict config to define standardSplitSizeInBytes in FTE` has all the changes in this section while we should have only changes related to `fault-tolerant-execution-target-task-split-count`/`fault-tolerant-execution-standard-split-size`

Reply:
Yeah I messed up... Let me fix this.